docs/buildah-config.1.md

```diff
@@ -77,6 +77,28 @@ ignore the `cmd` value of the container image.  However if you use the array
 form, then the cmd will be appended onto the end of the entrypoint cmd and be
 executed together.
 
+Note: The string form will be added executed via `sh -c`, where the array form
+will not use the containers shell.
```

Comment:
`added executed` is missing a word in the between them?  `containers` should be `container's`